sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCopilotPromoOptInStatus.md

```diff
@@ -0,0 +1,53 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Set-SPOCopilotPromoOptInStatus
+applicable: SharePoint Online
+title: Set-SPOCopilotPromoOptInStatus.md
+schema: 
+author: swathi.iruvanti
+ms.author: siruvanti
+ms.reviewer:
+---
+# Set-SPOCopilotPromoOptInStatus
+
+## SYNOPSIS
+
+Sets the Opt-In Copilot promo status for the tenant.
+
+## SYNTAX
+```powershell
+Set-SPOCopilotPromoOptInStatus -IsCopilotPromoStatusEnabled $true
+```
+## DESCRIPTION
+
+This cmdlet sets the Opt-In Copilot promo status for the tenant to `True` or `False`. The user must be a SharePoint Admin to run the cmlets.
```

Comment:
> The user must be a SharePoint Admin to run the cmlets. [](http://example.com/codeflow?start=85&length=55)

remove this sentence as it doesn't add any value.

SPO PowerShell cmdlets can only be called by SPO admins